packages/govuk-frontend/src/govuk/components/table/_index.scss

```diff
@@ -9,13 +9,15 @@
     border-collapse: collapse;
   }
 
-  // Modifier for tables with a lot of data. Tables with lots of data benefit
-  // from a smaller font size on small screens.
-  .govuk-table--small-text-until-tablet {
-    @include govuk-font-size($size: 16);
+  @if $govuk-new-typography-scale {
```

Comment:
The guidance PR introduces the `--small-text-until-tablet` modifier

But we haven't explained that it's opt-in only via the `$govuk-new-typography-scale` flag, should we?

Reply:
I'm aware that we don't clarify this and we should. I've got a plan to hold off on looking into this because we're planning on looking at how to do this more broadly via [the typography section restructure work](https://github.com/alphagov/govuk-design-system/pull/3343). Whatever we decide there I'll just copy for that guidance PR.